patches/server/0948-Improve-logging-and-errors.patch

```diff
@@ -0,0 +1,44 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Wed, 14 Dec 2022 15:52:11 -0800
+Subject: [PATCH] Improve logging and errors
+
+
+diff --git a/src/main/java/net/minecraft/server/packs/PathPackResources.java b/src/main/java/net/minecraft/server/packs/PathPackResources.java
+index 0232c29d96e1021a9f5a9678996993dc55fe7254..748c959c1e112279c361889be0c70f8ba81999eb 100644
+--- a/src/main/java/net/minecraft/server/packs/PathPackResources.java
++++ b/src/main/java/net/minecraft/server/packs/PathPackResources.java
+@@ -105,6 +105,12 @@ public class PathPackResources extends AbstractPackResources {
+         try (DirectoryStream<Path> directoryStream = Files.newDirectoryStream(path)) {
+             for(Path path2 : directoryStream) {
+                 String string = path2.getFileName().toString();
++                // Paper start
++                if (!Files.isDirectory(path2)) {
++                    LOGGER.error("Invalid directory entry: {} in {}. This will cause way more errors in vanilla.", string, this.root, new java.nio.file.NotDirectoryException(string));
```

Comment:
> This will cause way more errors in vanilla.

not fond of the wording of that, otherwise, lgtm

Reply:
k, just removed that.